app/assets/stylesheets/Edit_Patient/edit_patient.scss

```diff
@@ -0,0 +1,111 @@
+@import '../home/components/variables';
+
+.container{
+    display: flex;
+    flex-direction: column;
+    align-content: center;
+    justify-content: center;
+    .profile_picture{
+        text-align: center;
+        flex-basis: 100%;
+        .patient-info_img{
+            border-radius: 95px;
+            margin: 20px 0;
+            width: 160px;
+        }
+    }
+    
+    .patient-edit{
+        @extend %flex-column;
+        align-items: center;
+        margin: 20px 0;
+        padding: 20px;
+        &_field{
+            background-color: #66C0DD;
```

Comment:
add this as a variable in `/home/components/variables` that you already are loading..